ts/receiver/contentMessage.ts

```diff
@@ -317,7 +319,7 @@ function shouldDropBlockedUserMessage(content: SignalService.Content): boolean {
   if (!isMessageDataMessageOnly) {
     return true;
   }
-  const data = content.dataMessage;
+  const data = content.dataMessage as SignalService.DataMessage; // forcing it as we do know this field is set based on last line
```

Comment:
Do you mean because we use isMessageDataMessageOnly?

Reply:
yeah that's what I meant but it doesn't actually check for undefined.
Updated with an early return to true (message dropped) if the message has no dataMessage at all.
(if there is no data message, it cannot be a control message)

Reply:
Cool